meetings/2017-06-22.md

```diff
@@ -0,0 +1,123 @@
+# Node.js Foundation Community Committee Meeting 2017-06-22
+ 
+## Links
+ 
+* **GitHub Issue**: [community-committee#80](https://github.com/nodejs/community-committee/issues/80)
+* **Minutes Google Doc**: https://docs.google.com/document/d/14Pd6e40n1wPEuRBknDJUFwlf9XDhWI94jCgMEvTfVS4/
+* _Previous Minutes Google Doc_:
+https://docs.google.com/document/d/1J14T5lXuFWsaTqfkpmPruAZ4E-WsVe2V5TAaKwDjHN0/
+* _Video Recording_: https://www.youtube.com/watch?v=gwSWLXD2nLw 
```

Comment:
Yeah, I just kinda randomly picked one when creating it. There was pretty close to zero thought put into it ;-)